app/src/main/java/com/bogdan/codeforceswatcher/features/MainActivity.kt

```diff
@@ -216,7 +216,7 @@ class MainActivity : AppCompatActivity() {
 
         USERS(R.string.empty, R.id.navUsers),
         CONTESTS(R.string.contests, R.id.navContests),
-        ACTIONS(R.string.news, R.id.navActions),
+        ACTIONS(R.string.news, R.id.navNews),
```

Comment:
Probably you need to rename all `Actions` to `News` ?

Reply:
good point